src/main/java/seedu/address/model/person/Payment.java

```diff
@@ -0,0 +1,56 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+
+/**
+ * Represents a Student's payment status in the address book.
+ */
+public class Payment {
+
+    public final Boolean hasPaid;
+
+    /**
+     * Constructs an {@code Address}.
+     *
+     * @param hasPaid Boolean for payment status
+     */
+    public Payment(Boolean hasPaid) {
+        requireNonNull(hasPaid);
+        this.hasPaid = hasPaid;
+    }
+
+    /**
+     * Constructs an {@code Address}.
+     * Set to false as default
+     */
+    public Payment() {
+        this.hasPaid = false;
+    }
+
+    @Override
+    public String toString() {
+        return hasPaid ? "Paid" : "Not Paid";
```

Comment:
I forgot, is this ok for coding standard?

Reply:
I checked, as for coding standard the ternary operation is ok. 
But I will create a temporary variable to store it and then return the temporary variable.
Thanks for the spot

Reply:
Nevermind I think currently its the standard :open_mouth: